libs/unity/library/Runtime/Scripts/Media/IAudioSource.cs

```diff
@@ -10,55 +10,13 @@ namespace Microsoft.MixedReality.WebRTC.Unity
     /// Unity event corresponding to a new audio stream being started.
     /// </summary>
     [Serializable]
-    public class AudioStreamStartedEvent : UnityEvent<IAudioSource>
+    public class AudioStreamStartedEvent : UnityEvent<IAudioTrack>
     { };
 
     /// <summary>
     /// Unity event corresponding to an on-going audio stream being stopped.
     /// </summary>
     [Serializable]
-    public class AudioStreamStoppedEvent : UnityEvent<IAudioSource>
+    public class AudioStreamStoppedEvent : UnityEvent<IAudioTrack>
     { };
-
-    /// <summary>
-    /// Interface for audio sources plugging into the internal peer connection API to
-    /// expose a single audio stream to a renderer (<see cref="VideoRenderer"/> or custom).
-    /// </summary>
-    public interface IAudioSource
```

Comment:
This and its video sibling are now somehow merged with `WebRTC.IxTrack` (which probably makes sense to rename to `IxSource` since it should apply to local sources too).

There's some doc editing work to do, but the basic structure of this should be done.